authentication/oauth2-code/python/sdk/app.py

```diff
@@ -0,0 +1,14 @@
+from visier_api_analytic_model import DataModelApi
+
+# Alternative way to read config and create api client explicitly:
+# ```python
+# from visier_api_core import ApiClient
+# from visier_api_core.configuration import *
+# api_config = Configuration.from_env()
+# api_client = ApiClient(api_config)
+# data_model_api = DataModelApi(api_client)
+# ```
+
+data_model_api = DataModelApi()
```

Comment:
This would work with a (non-sourced) dotenv file as well right?

Reply:
I've added load_dotenv command, and now it will work.